integrations/google-analytics/lib/index.js

```diff
@@ -647,7 +647,7 @@ GA.prototype.pushEnhancedEcommerce = function(track, opts, trackerName) {
   var args = reject([
     self._trackerName + 'send',
     'event',
-    track.category() || 'EnhancedEcommerce',
```

Comment:
We could put this change behind a setting so it is opt in. That seems safer but this seems correct anyways?

Reply:
@CarlosMecha Curious if you have any concerns about this. I think this is the desired functionality but its possible customers are relying on the existing behavior.

Reply:
That makes sense, let's create a new setting!